.github/pull-request-template.md

```diff
@@ -0,0 +1,15 @@
+## Summary
+<!-- A brief summary of the PR -->
+
+## Checklist
+<!-- [ ] = no, [x] = yes -->
+
+ - [ ] I have searched for similar/duplicate PR's, and there are none.
+ - [ ] This is a **documentation update** (main code base unchanged).
+ - [ ] This is a **source code update** (main code base changed).
+   - [ ] I have created unit tests for the modified code base.
+   - [ ] All tests pass (if not please elaborate).
+   - [ ] This is a breaking change (requires major version update).
```

Comment:
I like the idea of having a checklist like this. However, I do not really like how it appears on the PR tab and in mentioned issues. It shows it as a task list when it really isn't, which could be a bit deceiving. Any ideas on how this could be prevented/changed. See the attached screenshot below to see what I mean.

![Cap 07-13-21 14 44 44](https://user-images.githubusercontent.com/35872177/125507895-03c96e0b-eacc-4266-8a7c-a3eb2ffdf57e.png)

Reply:
ahhhh i see what you mean. i will look at this. i can just handle the other 2 changes with this one too.